tests/Doctrine/Tests/Models/ECommerce/ECommerceCategory.php

```diff
@@ -85,7 +85,7 @@ public function getProducts()
         return $this->products;
     }
 
-    private function setParent(ECommerceCategory $parent)
+    public function setParent(ECommerceCategory $parent)
```

Comment:
It can be solved I think but too much work for this special case.

I hope it would be possible to remove the whole `UnusedPrivateElements` sniff someday in favour of PHPStan.